.github/workflows/code-quality-pr.yaml.yaml

```diff
@@ -0,0 +1,27 @@
+# This workflow finds which files was changed, print them, and then
+# run `pre-commit` on those files.
```

Comment:
```suggestion
# This workflow finds which files were changed, prints them, 
# and runs `pre-commit` on those files.
```

Reply:
done in https://github.com/ashleve/lightning-hydra-template/pull/364/commits/479def86850bfe89ca07d1e7a30eb83c5a5f640f